docs/concepts/extensions.md

```diff
@@ -18,6 +18,36 @@ Out-of-the-box components are configurable, or altogether new components can be
 - [Endpoints](/guides/api-endpoints)
 - [Hooks](/guides/api-hooks)
 
+
+## Usefull tips and tricks
+
+Nodemon
+
+Nodemon can be used for automatic reloading.
+Basicly whenever you change a hook or another extension, your directus instance will automaticly restart.
+
+Setup
+
+	1. install nodemon: npm i -g nodemon kill-port
+	2. go to the package.json of your directus project.
+	3. Add the following configuration:
+	
+  "nodemonConfig": {
+    "ignore": [
+      "node_modules/*"
+    ],
+    "watch": [
+      "extensions/"
+    ],
+    "events": {
+      "restart": "kill-port 8055",
```

Comment:
This doesn't feel like the proper way to restart the Directus server. `kill-port` will find the process, and force-quit it. This means that things like database connections aren't properly closed, and might cause other problems.

Another way that should work is relying on `ts-node-dev` (more or less nodemon for typescript) and add a custom filepath watcher to your extensions folder, that'll ensure the node process is properly restarted instead of killing the process.